scriptmodules/supplementary/splashscreen.sh

```diff
@@ -137,7 +159,7 @@ function choose_splashscreen() {
             options+=("$i" "$splashdir")
             ((i++))
         fi
-    done < <(find "$path" -type f ! -regex ".*/\..*" ! -regex ".*LICENSE" ! -regex ".*README.*" | sort)
+    done < <(find "$path" -type f ! -regex ".*/\..*" ! -regex ".*LICENSE" ! -regex ".*README.*" ! -regex ".*.sh" | sort)
```

Comment:
it should be `".*\.sh"` as your regexp would match a file called "whoosh" for example (works anyway, but technically wrong).